ooyala_player/ooyala_player.py

```diff
@@ -196,6 +197,8 @@ def student_view(self, context):
             'height': self.height,
             'autoplay': self.autoplay,
             'config_url': json_config_url,
+            'publish_completion_url': self.runtime.handler_url(self, 'publish_completion', None).rstrip('/?'),
```

Comment:
This is creating a URL that ends with a literal `"/None"`.  It works, but it's ugly, and a little fragile.  Can you change it to `self.runtime.handler_url(self, 'publish_completion', '')`?  

Probably handler_url should deal with None in a better way, but that seems like a more risky change, and not worth the benefit for this PR.

Reply:
I agree.